ArduPlane/Log.cpp

```diff
@@ -232,7 +234,9 @@ void Plane::Log_Write_AETR()
         ,throttle : SRV_Channels::get_output_scaled(SRV_Channel::k_throttle)
         ,rudder   : SRV_Channels::get_output_scaled(SRV_Channel::k_rudder)
         ,flap     : SRV_Channels::get_slew_limited_output_scaled(SRV_Channel::k_flap_auto)
-        ,speed_scaler : get_speed_scaler(),
+        ,speed_scaler_x : get_rpy_speed_scaler().x
```

Comment:
we should fetch it once with:
 const Vector3f &ss = get_rpy_speed_scaler()

Reply:
done